docs/repl.md

```diff
@@ -17,7 +17,31 @@ This would open a Chrome browser that you can control with the REPL interface. M
 wdio repl chrome -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY
 ```
 
-You can apply any available options for your REPL session. (See `wdio --help` for options.)
+```sh
+wdio repl android
+OR
+wdio repl ios
+```
+
+This would open Chrome/Safari session on connected device/emulator/simulator. Make sure Appium running on port `4444` in order to initiate the session.
+
+```sh
+wdio repl './path/to/your_app.apk'
+```
+
+This would open App session on connected device/emulator/simulator. Make sure Appium running on port `4444` in order to initiate the session.
+
+Capabilities for iOS device can be passed with arguments:
+-v      - platformVersion
+-d      - deviceName
+-u      - udid (for real devices)
+
+USAGE:
+`wdio repl ios --platformVersion 11.3 --deviceName 'iPhone 7' --udid 123432abc`
```

Comment:
```suggestion
``'sh
$ wdio repl ios --platformVersion 11.3 --deviceName 'iPhone 7' --udid 123432abc
``'
```

__Note:__ I used `'` as last character so that I can make this suggesiton, please replace is with a proper up tick